lib/galaxy/__init__.py

```diff
@@ -91,5 +91,3 @@ class NullHandler( logging.Handler ):
         def emit( self, record ):
             pass
     logging.NullHandler = NullHandler
-
-import galaxy.eggs
```

Comment:
Copy-pasting from my comment on the commit for easier review:

This line was fixing the flake8 error: `lib/galaxy/__init__.py:95:1: F401 'galaxy' imported but unused` .

Unfortunately this is breaking Galaxy datatypes loading with a ton of errors like:

```
galaxy.datatypes.registry ERROR 2015-07-07 11:19:12,332 Error importing datatype module galaxy.datatypes.binary: 'module' object has no attribute 'galaxy'
Traceback (most recent call last):
  File "/usr/users/ga002/soranzon/software/bgruening_galaxy/lib/galaxy/datatypes/registry.py", line 204, in load_datatypes
    module = getattr( module, mod )
AttributeError: 'module' object has no attribute 'galaxy'
```

@dannon: suggestions? Should we add a `# noqa` at the end of this line instead?